tasks/2-code-sandbox/README.md

```diff
@@ -103,8 +103,8 @@ can be found at **https://preview.golem.network/**
 
 ## Useful links:
 
-- Javascript docs: https://docs.golem.network/creators/javascript/quickstart/
-- Dapps docs: https://docs.golem.network/creators/dapps/ 
+- Javascript docs: https://docs.golem.network/docs/creators/javascript/quickstart/
```

Comment:
Broken link, should be https://docs.golem.network/docs/creators/javascript/quickstarts/quickstart